src/api-spec/buf.yaml

```diff
@@ -1,8 +1,8 @@
 version: v1
 name: buf.build/tdex-network/tdex-daemon
+breaking:
```

Comment:
you need deps here, pinting to `buf.build/tdex-network/tdex-daemon`

Reply:
https://github.com/tdex-network/tdex-daemon/blob/master/api-spec/protobuf/buf.yaml#L3

Reply:
I need to keep `buf.build/tdex-network/tdex-protobuf` in order to get `types.proto` right?
What about `buf.build/googleapis/googleapis` ?

Reply:
no need for google API I guess, used only in trade.proto stuff